### PR TITLE
Add Firefox Wires driver support, update selenium-server version and refactor code

### PR DESCRIPTION
**Pull request by SkReD**

### CODE REVIEW

**Comment by jsdevel** [comment]
can you resolve the conflicts?

**Comment by SkReD** [comment]
Resolved

**Comment by SkReD** [comment]
Tests failed..i'll try to fix it as soon as possible

File: lib/config.js [hunk 1]
```diff
@@ -47,6 +47,24 @@ module.exports = {
...
+      download: {
+        name: 'wires-0.6.2-'
+        + ( isWin
+                ? 'win.zip'
+                : ( isMac
+                    ? 'OSX.gz'
+                    : 'linux64.gz'
+            )
+        ),
+        url:'https://github.com/jgraham/wires/releases/download/v0.6.2/'
```
Comment (jsdevel) [inline lib/config.js]: is this an official repo?

Comment (SkReD) [inline lib/config.js]: Taked from here https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver
Link from selenium download page also target to mdn docs

File: lib/init-environment.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+var NO_HOME_VAR_FOUND_IN_ENV=1;
+var INVALID_ENV_PATH_LOCATION=2;
+var home = process.env.HOME || process.env.USERPROFILE;
+var SELENIUM_BINARIES_HOME = process.env.SELENIUM_BINARIES_HOME;
+
+if(process.env.SELENIUM_BINARIES_HOME){
+	if(!isDir.sync(process.env.SELENIUM_BINARIES_HOME)){
+		cli.error('SELENIUM_BINARIES_HOME was set to a path that does not exist!');
+		cli.exit(INVALID_ENV_PATH_LOCATION);
```
Comment (jsdevel) [inline lib/init-environment.js]: can you update your tab settings?

Comment (SkReD) [inline lib/init-environment.js]: updated

**Comment by jsdevel** [comment]
@SkReD couple of things:

1. Can you split this up into multiple PRs?  Specifically, I'd like to review any refactoring separately from feature additions.
2. Can you limit the commit count to 1 for each PR?
3. Can you set your github user details locally?  I'm not seeing your username in your git commits.

Thanks!

**Comment by SkReD** [comment]
@jsdevel ok, i'll try. I don't do any big pull requests in projects on github, so i'm sorry for this little chaos.

**Comment by jsdevel** [comment]
@SkReD no worries!  Sometimes it's difficult, especially when refactoring.

**Pull request closed without merging**